var/spack/repos/builtin/packages/pfunit/package.py

```diff
@@ -17,7 +17,13 @@ class Pfunit(CMakePackage):
 
     maintainers = ['citibeth']
 
-    version('3.2.9', 'e13d8362284b13b7c863e2fe769a9d5c')
+    # version('4.0.0',  sha256='b8b6470f2b1e2b19c164c244c10e803bd69c8da9a6a5a65ba7c479fb8b92a1e1')
+    version('3.3.3',  sha256='9f673b58d20ad23148040a100227b4f876458a9d9aee0f0d84a5f0eef209ced5')
+    version('3.3.2',  sha256='b1cc2e109ba602ea71bccefaa3c4a06e7ab1330db9ce6c08db89cfde497b8ab8')
+    version('3.3.1',  sha256='f8f4bea7de991a518a0371b4c70b19e492aa9a0d3e6715eff9437f420b0cdb45')
+    version('3.3.0',  sha256='4036ab448b821b500fbe8be5e3d5ab3e419ebae8be82f7703bcf84ab1a0ff862')
+    version('3.2.10', sha256='b9debba6d0e31b682423ffa756531e9728c10acde08c4d8e1609b4554f552b1a')
+    version('3.2.9',  sha256='403f9a150865700c8b4240fd033162b8d3e8aeefa265c50c5a6fe14c455fbabc')
```

Comment:
No need to change hash.  Please just use old MD5 for 3.2.9, then nobody has to verify.

Reply:
Happy to do so. But spack is switching to sha256. Do you want to delay the switch for 3.2.9?

Reply:
This change is OK.
```
gs611-tyndall:Downloads rpfische$ shasum -a 256 3.2.9
403f9a150865700c8b4240fd033162b8d3e8aeefa265c50c5a6fe14c455fbabc  3.2.9
gs611-tyndall:Downloads rpfische$ md5 3.2.9
MD5 (3.2.9) = e13d8362284b13b7c863e2fe769a9d5c
```

Reply:
For crosschecking:

$ sha256sum pFUnit-4.0.0.tar.gz
b8b6470f2b1e2b19c164c244c10e803bd69c8da9a6a5a65ba7c479fb8b92a1e1  pFUnit-4.0.0.tar.gz

$ sha256sum pFUnit-3.3.3.tar.gz
9f673b58d20ad23148040a100227b4f876458a9d9aee0f0d84a5f0eef209ced5  pFUnit-3.3.3.tar.gz

$ sha256sum pFUnit-3.3.2.tar.gz
b1cc2e109ba602ea71bccefaa3c4a06e7ab1330db9ce6c08db89cfde497b8ab8  pFUnit-3.3.2.tar.gz

$ sha256sum pFUnit-3.3.1.tar.gz
f8f4bea7de991a518a0371b4c70b19e492aa9a0d3e6715eff9437f420b0cdb45  pFUnit-3.3.1.tar.gz

$ sha256sum pFUnit-3.3.0.tar.gz
4036ab448b821b500fbe8be5e3d5ab3e419ebae8be82f7703bcf84ab1a0ff862  pFUnit-3.3.0.tar.gz

$ sha256sum pfunit-3.2.9.tar.gz
403f9a150865700c8b4240fd033162b8d3e8aeefa265c50c5a6fe14c455fbabc  pfunit-3.2.9.tar.gz

$ sha256sum pFUnit-3.2.10.tar.gz
b9debba6d0e31b682423ffa756531e9728c10acde08c4d8e1609b4554f552b1a  pFUnit-3.2.10.tar.gz